src/main/java/jenkins/plugins/slack/ActiveNotifier.java

```diff
@@ -134,7 +134,7 @@ public void completed(AbstractBuild r) {
         if (null != previousBuild) {
             do {
                 previousBuild = previousBuild.getPreviousCompletedBuild();
-            } while (null != previousBuild && previousBuild.getResult() == Result.ABORTED);
+            } while ((null != previousBuild && previousBuild.getResult() == Result.ABORTED) || previousBuild.getNumber() == r.getNumber());
```

Comment:
seems ok, have you tested it out?

Reply:
Yep, I was able to reproduce the error on the local Jenkins instance created by running the plugin.  I confirmed that when I made my change the notification properly showed up as well as logging the last build properly, fixing #568 too.

I'll check out the failing build and push more changes if need be.